### ISSUE TITLE
If you miss the "unverified session" toast, it's very unclear how to verify the new session.

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#12857]
I assume I need to reopen my memberinfo in an e2e room and then find the new login and verify it?  This is a *very* unintuitive thing to do - I'd ideally expect to find it in Settings: Security rather than thinking to look in my own MemberInfo?  Or at least some copy or a link from there to take you to the right place?

**Comment by bwindels** [comment]
cc @nadonomy

**Comment by nadonomy** [comment]
This is a casualty of us descoping Settings from the phases to date. Should we re-assign this to the same phase Settings is in, or close it given the work is already captured?

**Comment by jryans** [comment]
I'll mark this as after release for now.

**Comment by jryans** [comment]
Moved to the separate story about device mgmt.

**Comment by ara4n** [comment]
for now can “later“ show you a popup telling you to check your memberinfo to resume?

**Comment by jryans** [comment]
> for now can “later“ show you a popup telling you to check your memberinfo to resume?

It could, or maybe the toast itself could state something about this...? Over to @nadonomy for design thoughts.

**Comment by jryans** [comment]
> I wonder if we should find a way to label so we don't need to remember arbitrary numbers.

Yes, the number labels are too mysterious, I think most people agree. Filed https://github.com/vector-im/feature-dashboard/issues/72 to find a better way.

**Comment by TheJJ** [comment]
The list of sessions contains information like ID, name and last seen, but no button to perform the trusting procedure or a display of the trust state. There should be both for each device.
